code/obj/item/clothing/injector_mask_belt.dm

```diff
@@ -427,3 +406,153 @@ There's A LOT of duplicate code here, which isn't ideal to say the least. Should
 	check_trigger(mob/M)
 		if(isdead(M)) return 1
 		else return 0
+
+/proc/autoinjector_trigger_names(var/setType)
+	var/list/cond_types = concrete_typesof(/datum/injector_belt_condition)
+	. = new/list()
+
+	if (!setType)
+		. += "None"
+
+	for(var/A in cond_types)
+		var/datum/injector_belt_condition/C = new A()
+		. += C.name
+		if (setType)
+			.[C.name] = A
+
+/proc/autoinjector_ui_data(var/datum/reagents/R, var/datum/injector_belt_condition/condition, var/min_time, var/inj_amount)
+	. = list()
+
+	var/list/reagentData = null
+	if (R)
+		reagentData = list(
+			maxVolume = R.maximum_volume,
+			totalVolume = R.total_volume,
+			contents = list(),
+			finalColor = "#000000"
+		)
+
+		var/list/contents = reagentData["contents"]
+		if(istype(R) && R.reagent_list.len>0)
```

Comment:
```suggestion
		if(istype(R) && (length(R.reagent_list) > 0))
```
per the readme's code style guide, `.len` is not to be used for getting the length of lists